uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Helper/ContactActionsMenu/WorkflowActionGroup.php

```diff
@@ -212,18 +191,20 @@ public function getManageLineManagerButton() {
    * Returns an instance of an ActionsGroupButtonItem
    *
    * @param array $params
-   * @param string $onClick
+   * @param array $attributes
    *
    * @return ActionsGroupButtonItem
    */
-  private function getMenuButton($params, $onClick = '') {
+  private function getMenuButton($params, $attributes = []) {
```

Comment:
what about changing this to `private function getMenuButton($params, array $attributes = [])` ? This way, we force this to always be an array, and then you won't need the `if ($attributes) ` statement below